android/src/main/java/com/applovin/reactnative/AppLovinMAXAdViewManager.java

```diff
@@ -75,4 +75,12 @@ public void onDropViewInstance(@NotNull AppLovinMAXAdView view)
 
         super.onDropViewInstance( view );
     }
+
+    @Override
+    public void onAfterUpdateTransaction(final AppLovinMAXAdView view)
```

Comment:
Nit, but we usually order methods in the order which they occur (this is especially true with ad lifecycle callbacks) to make it easier to understand the logical flow. Maybe this should be swapped with `onDropViewInstance(...)`?

Reply:
Right, updated the order.